hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java

```diff
@@ -3822,5 +3822,10 @@ public long getLastDirScannerFinishTime() {
   public void setLastDirScannerFinishTime(long time) {
     this.lastDirScannerFinishTime = time;
   }
+
+  @Override
+  public long getPendingAsyncDiskOperations() {
```

Comment:
Thanks Sir @Hexiaoqiao help review.

Because current async disk operation include deletion of block files and requesting sync_file_range(),
so consider metrics name is  `PendingAsyncDiskOperations`

Reply:
I see, However we expose `PendingAsyncDiskOperations` to metric but invoke `countPendingDeletions`, it could be confused to other devs. Suggest to unify the define at both side if changes. Thanks.

Reply:
Thanks  @Hexiaoqiao  for your suggestions.

You are right, the method name definitions at both sides need to be consistent.
From the current implementation, it might be better to changing countPendingDeletions to countPendingAsyncDiskOperations, what do you think about?